types/steam-user/index.d.ts

```diff
@@ -211,25 +205,24 @@ declare class SteamUser extends EventEmitter {
      * @param [callback] - Called when an activation SMS has been sent.
      */
     enableTwoFactor(
-        callback?: (err: Error | null, response: Record<string, any>) => void,
-    ): Promise<Record<string, any>>;
+        callback?: (err: Error | null, response: TwoFactorResponse) => void,
+    ): Promise<TwoFactorResponse>;
 
     /**
      * Finalize the process of enabling TOTP two-factor authentication
      * @param secret - Your shared secret
      * @param activationCode - The activation code you got in your email
      * @param [callback] - Called with a single Error argument, or null on success
      */
-    finalizeTwoFactor(secret: Buffer, activationCode: string, callback?: (err: Error | null) => void): Promise<void>;
+    finalizeTwoFactor(secret: string, activationCode: string, callback?: (err: Error | null) => void): Promise<void>;
```

Comment:
I have never used this function, but this is where `secret` is consumed: https://github.com/DoctorMcKay/node-steam-user/blob/d24569d6041b63e1fc36a1ab21a4becd2cec2b48/components/twofactor.js#L61
(If you are sure that the function does not accept a `Buffer`, you should also update the comment some lines above in https://github.com/DoctorMcKay/node-steam-user/pull/480)

It uses [DoctorMcKay/node-steam-totp](https://github.com/DoctorMcKay/node-steam-totp) under the hood, which is documented to accept either `string` or `Buffer`: https://github.com/DoctorMcKay/node-steam-totp/blob/359c2d57b480c0808474416367530797e64ef773/index.js#L20

Depending on the outcome of https://github.com/DoctorMcKay/node-steam-user/pull/480, maybe it would be better to type it as `string | Buffer`? If you tested it, it might as well be wrong documentation over there, but let's wait for an answer maybe.

Reply:
I tried using Buffer and it was showing error that code is not correct and only then I used string and that worked perfect

Reply:
Also I just checked ``SteamTotp.generateAuthCode()`` secret param is string type, not Buffer

Reply:
I made tests and this is correct types for this param, I think we will wait DoctorMcKay/node-steam-user#480 too long (there is 6 more PR without replies) to merge.

Reply:
Alright, sounds reasonable. I still think your PR over there should include a change on [line 40](https://github.com/DoctorMcKay/node-steam-user/blob/d24569d6041b63e1fc36a1ab21a4becd2cec2b48/components%2Ftwofactor.js#L40) as well though :)